lib/components/chat_history/twitch/cheer_event.dart

```diff
@@ -0,0 +1,65 @@
+import 'package:flutter/material.dart';
+import 'package:flutter/rendering.dart';
+import 'package:intl/intl.dart';
+import 'package:provider/provider.dart';
+import 'package:rtchat/models/messages/twitch/event.dart';
+import 'package:rtchat/models/style.dart';
+
+class TwitchCheerEventWidget extends StatelessWidget {
+  final TwitchCheerEventModel model;
+  final NumberFormat _formatter = NumberFormat.compactSimpleCurrency();
```

Comment:
Format as a number instead of currency.

Reply:
I removed the formatter, and left the `bits` as an `int`.